driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java

```diff
@@ -120,31 +124,33 @@ public void run() {
 
         Executor initExecutor = manager.cluster.manager.configuration.getPoolingOptions().getInitializationExecutor();
 
-        ListenableFuture<List<Void>> allConnectionsFuture = Futures.allAsList(connectionFutures);
+        ListenableFuture<List<Void>> allConnectionsFuture = Futures.successfulAsList(connectionFutures);
```

Comment:
One catch here is that if the *first* connection we try to open fails, the host's connection count goes back to 0 and the host is marked down immediately.
However this only happen for the initial connection. For reconnection attempts, we know that a connection has been opened by the reconnection handler before attempting to create pools.
So I don't think we need to be too clever here. The alternative would be to wait until all connections have been attempted before marking down, which sounds too convoluted.